utilities/transactions/pessimistic_transaction_db.h

```diff
@@ -101,9 +101,21 @@ class PessimisticTransactionDB : public TransactionDB {
                                     const std::string& column_family_name,
                                     ColumnFamilyHandle** handle) override;
 
+  virtual Status CreateColumnFamilies(
```

Comment:
Keep `override` and remove `virtual`, per https://releases.llvm.org/9.0.1/tools/clang/tools/extra/docs/clang-tidy/checks/modernize-use-override.html.